src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2521,6 +2521,14 @@
     text: az vmss wait --updated --instance-id 1 --name MyScaleSet --resource-group MyResourceGroup
 """
 
+helps['vmss set-orchestration-service-state'] = """
+type: command
+short-summary: Change ServiceState property for a given service within a VMSS.
+examples:
+  - name: Change ServiceState property for AutomaticRepairs
+    text: az vmss set-orchestration-service-state --service-name AutomaticRepairs --action Resume --name MyScaleSet --resource-group MyResourceGroup
```

Comment:
how to get the state? This will impact command name, will it be orchestration-service-state set or set-orchestration-service-state

Reply:
As I know, currently there is only one set operation for it. The result of this operation will show in the instance view. We can use `az vmss get-instance-view -g rg -n vmss_name`  to get the orchestration services state. This is also another request they asked in the email. It is also supported in this PR.

Reply:
thanks for clarification. As long as there will no command on show/list etc, the command should be ok. Another question, I remembered that automatic instance repair command is preview, what's status of this command?

Reply:
we have already remove the Preview tag as they asked for this before.

Reply:
yes. should we mark this as preview also?

Reply:
I will send an email to get their confirmation first.

Reply:
confirmed not to add Preview tag.